onchain/Dockerfile

```diff
@@ -11,16 +11,16 @@
 # the License.
 
 # syntax=docker.io/docker/dockerfile:1.4
-FROM node:16.17.0-alpine3.15 as base
+FROM node:16.17.0-bullseye as base
```

Comment:
Should we bump the node version?

I suggest using, at least, the latest LTS version, which is 18.16.